op-node/rollup/attributes/engine_consolidate.go

```diff
@@ -125,13 +135,32 @@ func checkEIP1559ParamsMatch(opCfg *params.OptimismConfig, attrParams *eth.Bytes
 			translated = true
 		}
 
-		bd, be := eip1559.DecodeHoloceneExtraData(blockExtraData)
+		// Decode block parameters and check for mismatch
+		var (
+			bd, be   uint64
+			bm       uint8
+			extraErr string
+		)
+		if isJovian {
+			bd, be, bm = eip1559.DecodeMinBaseFeeExtraData(blockExtraData)
+		} else {
+			bd, be = eip1559.DecodeHoloceneExtraData(blockExtraData)
+		}
+
 		if ad != bd || ae != be {
```

Comment:
We're missing a check to ensure that the minimum base fee in the block header matches the minimum base fee in the configuration. This function is called in `AttributesMatchBlock` where `attrs.EIP1559Params` is passed in as `attrParams`, but we're not passing in the new `attrs.MinBaseFeeLog2` to check against `bm` as well.